bpf-gpl/xdp.c

```diff
@@ -0,0 +1,93 @@
+#include <linux/if_ether.h>
```

Comment:
This file is only for showing the structure of the XDP program. Won't be included in the final PR.

Reply:
I think it could be a good idea to include this.  For reference, and so that we can evolve it later to what we really want.  But please add a comment at the top to say that it is not yet in active use.

Reply:
Should also add similar copyright as we have in other files in this directory.